themes/custom/droopler_theme/scss/layout/_modifiers.scss

```diff
@@ -511,18 +510,54 @@ $d-p-paddings-mobile: map-merge(
       > .paragraph:not(.content-moved-inside),
       > .paragraph.content-moved-inside .content-inside-wrapper {
         padding-#{$placement}: $value;
+
+        @include media-breakpoint-down(sm) {
+          padding-#{$placement}: map-get($d-p-paddings-mobile, $size);
+        }
```

Comment:
tutaj taki nieduży refactor, wcześniej to było w osobnym bloku, który od tego różnił się tylko media-breakpointem dodanym na początku, o tak:

```
@include media-breakpoint-down(sm) {
  @each $size, $value in $d-p-paddings-mobile {
    @each $placement in (top, bottom) {
      .padding-#{$placement}-#{$size} {
        > .paragraph:not(.content-moved-inside),
        > .paragraph.content-moved-inside .content-inside-wrapper {
          padding-#{$placement}: $value;
        }
      }
    }
  }
}
```